package-lock.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
There is already a yarn.lock file, please use that. Yarn is preferred over npm for multiple reasons, you can also read, quite interesting.

Reply:
yes, sure